awpa/gram_py35/token.py

```diff
@@ -87,3 +87,5 @@ def ISNONTERMINAL(x):
 
 def ISEOF(x):
     return x == ENDMARKER
+
```

Comment:
This trailing whitespace is present upstream I guess, and `export-grammar.sh` didn't strip it.

I'm inclined to probably not do so by hand.